docker-compose.yml

```diff
@@ -0,0 +1,103 @@
+version: '3'
+services:
+  node-0:
+    image: cennznet/cennznet:1.1.1
```

Comment:
Is it temporary that most of them will run the previous version?

Reply:
Yeah, I'm thinking we'll update it this file with each release. For now I was using it to test upgrades from 1.1.1 -> 1.2.0